Doc/tutorial/venv.rst

```diff
@@ -98,8 +98,8 @@ Managing Packages with pip
 ==========================
 
 You can install, upgrade, and remove packages using a program called
-:program:`pip`.  By default ``pip`` will install packages from the Python
-Package Index, <https://pypi.org>.  You can browse the Python
+:program:`pip`.  By default ``pip`` will install packages from the `Python
+Package Index, <https://pypi.org>`_.  You can browse the Python
```

Comment:
Let's remove this comma:

<img width="570" alt="image" src="https://user-images.githubusercontent.com/1324225/216349838-06fc55d1-5947-4d4e-add7-3f7f6e4963ec.png">


```suggestion
Package Index <https://pypi.org>`_.  You can browse the Python
```

Reply:
Hello. Good catch, I'll remove it. Thanks.